pkg/actor/validate_version.go

```diff
@@ -312,6 +327,32 @@ func IsJobPodRunning(
 	return nil
 }
 
+func IsContainerStatusImagePullBackoff(
+	ctx context.Context,
+	clientset kubernetes.Interface,
+	job *kbatch.Job,
+	l logr.Logger,
+	image string,
+) error {
+	//get pod for the job we created
+	pods, err := clientset.CoreV1().Pods(job.Namespace).List(ctx, metav1.ListOptions{
```

Comment:
Should we check on the different error types from k8s?

Reply:
not sure about this, in our current workflow it does not help, but when we will make it generic for sure